supporting/shimexe/shim.cs

```diff
@@ -91,6 +91,12 @@ internal struct PROCESS_INFORMATION {
             if(!string.IsNullOrEmpty(cmd_args)) cmd_args = " " + cmd_args;
             var cmd = "\"" + path + "\"" + cmd_args;
 
+            foreach (KeyValuePair<string, string> entry in config) {
+                if (entry.Key.StartsWith("env::")) {
+                    Environment.SetEnvironmentVariable(entry.Key.Substring(5), Environment.ExpandEnvironmentVariables(entry.Value));
```

Comment:
Change substring accordingly
```suggestion
                    Environment.SetEnvironmentVariable(entry.Key.Substring(4), Environment.ExpandEnvironmentVariables(entry.Value));
```